packages/playground/website/vite.config.ts

```diff
@@ -80,9 +80,11 @@ export default defineConfig(({ command }) => {
 			ignoreWasmImports(),
 			virtualModule({
 				name: 'website-config',
-				content: `export const remotePlaygroundOrigin = ${JSON.stringify(
-					playgroundOrigin
-				)};`,
+				content: `
+				export const remotePlaygroundOrigin = ${JSON.stringify(playgroundOrigin)};
+				export const buildVersion = ${JSON.stringify(
+					(new Date().getTime() / 1000).toFixed(0)
+				)};					`,
```

Comment:
spacing?

Reply:
this seems pragmatic, but can we not get access to the `git` ref or some deterministic value through `vite`?

Reply:
well, if that doesn't slow it down much seems fine. this is only at build-time yeah? I think that'd be fine.

`git describe --always --long --dirty` can also indicate if there were modified files in the repo when it was built, which can be helpful to know when comparing to what you build locally.

if `git` is already installed this is great. if not, probably not worth it, but there's always `libgit2` and the corresponding NPM package for it.

Reply:
It can be lenient, too. Use `git` if available, use the current date otherwise.

Reply:
Done!